citesphere/src/main/java/edu/asu/diging/citesphere/core/zotero/impl/ZoteroManager.java

```diff
@@ -308,7 +360,56 @@ public ZoteroUpdateItemsStatuses updateCitations(IUser user, String groupId, Lis
         List<List<String>> ignoreFieldsList = new ArrayList<>();
         List<List<String>> validCreatorTypesList = new ArrayList<>();
         for (ICitation citation : citations) {
-            Item item = itemFactory.createItem(citation, citation.getCollections());
+            Item item = itemFactory.createItem(citation, citation.getCollections());            
+            items.add(item);
+            itemTypeFields = getItemTypeFields(user, citation.getItemType());
+            itemTypeFields.add(ZoteroFields.ITEM_TYPE);
+            itemTypeFields.add(ZoteroFields.CREATOR);
+            itemTypeFields.add(ZoteroFields.COLLECTIONS);
+            itemTypeFields.add(ZoteroFields.KEY);
+            itemTypeFields.add(ZoteroFields.VERSION);
+            List<String> ignoreFields = createIgnoreFields(itemTypeFields, item, true);
+            List<String> validCreatorTypes = getValidCreatorTypes(user, citation.getItemType());
+            ignoreFieldsList.add(ignoreFields);
+            validCreatorTypesList.add(validCreatorTypes);
+            
+            //Add metadata note to the list
+            Item metaData = itemFactory.createMetaDataItem(citation);
+            items.add(metaData);
+            if (metaData.getKey() != null && !metaData.getKey().isEmpty()) {
+                itemTypeFields = getItemTypeFields(user, ItemType.NOTE);
+                itemTypeFields.add(ZoteroFields.KEY);
+                itemTypeFields.add(ZoteroFields.VERSION);
+            } else {
+                itemTypeFields = getItemTypeFields(user, ItemType.NOTE);
+            }
```

Comment:
`itemTypeFields = getItemTypeFields(user, ItemType.NOTE);` repeats and doesn't need to be in the if statement.